charts/pega/values.yaml

```diff
@@ -149,9 +149,11 @@ global:
         # Inactivity time after which requestor is passivated
         passivationTimeSec: 900
 
-      service:
-        port: 7003
-        targetPort: 7003
+      # Port 7003 turned off by default in all installs except large.
+      # Uncomment following lines to expose service and ports.
```

Comment:
Port 7003 is not exposed by default in most deployments; the exception is large deployments.
To expose service and ports in your deployment, you must uncomment the following lines.

Reply:
What is the reason we want to keep port and target port for stream commented in main values.yaml and only to have them by default in large values.yaml

Reply:
In conversations with some customers, this is what we've found.

Most clients don’t use direct stream injection which is the reason for that ingress. The only real use case is in a CDH deployment (primarily larger deployments). Rather than make it available across all deploys, we make it available for the most applicable cases and be able to turn off and on fairly easily by customer if they want to.

Reply:
Please change the comment description as suggested by Taz.